songr/src/main/java/com/lab11java401d3/songr/controllers/BonusAlbumsController.java

```diff
@@ -0,0 +1,104 @@
+package com.lab11java401d3.songr.controllers;
+
+
+import com.lab11java401d3.songr.SongNotFoundException;
+import com.lab11java401d3.songr.database.Album;
+import com.lab11java401d3.songr.database.AlbumRepository;
+import com.lab11java401d3.songr.database.Song;
+import com.lab11java401d3.songr.database.SongRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.view.RedirectView;
+
+import java.util.List;
+import java.util.Optional;
+
+@Controller
+@RequestMapping("/bonusAlbums")
+public class BonusAlbumsController {
+
+    @Autowired
+    SongRepository songRepo;
+
+    @Autowired
+    AlbumRepository albumRepo;
+
+    @GetMapping
+    public String getSongs(Model model) {
+        List<Song> songs = songRepo.findAll();
+        List<Album> albums = albumRepo.findAll();
+
+        model.addAttribute("songs", songs);
+        model.addAttribute("albums", albums);
+
+        return "songs";

```

Comment:
This method returns an error because there is something wrong with your HTML in the template "songs"